qa/rolling-upgrade/src/javaRestTest/java/org/elasticsearch/upgrades/LogsIndexModeRollingUpgradeIT.java

```diff
@@ -69,6 +69,11 @@ protected String getTestRestCluster() {
           "data_stream": {},
           "priority": 500,
           "template": {
+            "settings": {
```

Comment:
@salvatore-campagna I had to make this change because of the index that get created before the upgrade starts would otherwise already in logsdb format. So I set index.mode=standard in `STANDARD_TEMPLATE` constant. I think this is the best way to test upgrading from standard to logsdb? (that works for all versions)